apps/andi/assets/css/data_dictionary_form.scss

```diff
@@ -15,21 +15,34 @@ $tick-height: 1rem + $field-padding - 0.44;
     padding: 2rem;
     margin-right: 2rem;
     margin-top: 1.7rem;
-    grid-template-areas: 
+    grid-template-columns: 33% 33% 33%;
+    grid-template-areas:
       "name . type"
-      "selector . item-type"
-      "description description description" 
+      "selector . type-info"
+      "description description description"
       "pii masked demographic"
-      "biased rationale ."; 
- 
+      "biased rationale .";
+
       .select, .input {
-        width: 90%; 
+        margin: 10px 0 10px 0;
```

Comment:
what's up with the pixels?

Reply:
Just matching what app.scss does for input fields